app/menu.js

```diff
@@ -245,8 +245,9 @@ function updateForMac(template, messages, options) {
   );
 
   // Replace Window menu
+  const windowIndex = options.includeSetup ? 4 : 3;
   // eslint-disable-next-line no-param-reassign
-  template[3].submenu = [
+  template[windowIndex].submenu = [
```

Comment:
Nice! 😄  https://github.com/signalapp/Signal-Desktop/blob/a3cf81b44a0ed4d1796f62dbfe9b29be8c568f50/app/menu.js#L279-L281

Reply:
Yep, this is a reimplementation, but it's ready for hotfix. :0)